classes/DirtEventTypes.sc

```diff
@@ -92,9 +95,9 @@ DirtEventTypes {
 					});
 				};
 
-				if(~progNum.notNil)   { var num = ~progNum;   schedmidi.value({ midiout.program(chan, num) })};
-				if(~midibend.notNil)  { var val = ~midibend;  schedmidi.value({ midiout.bend(chan, val)    })};
-				if(~miditouch.notNil) { var val = ~miditouch; schedmidi.value({ midiout.touch(chan, val)   })};
+				if(~progNum.notNil)   { var num = ~progNum;   donecmd.value(\program); schedmidi.value({ midiout.program(chan, num) })};
```

Comment:
I'm curious why this works:
```
donecmd.value(\program); schedmidi.value({ midiout.program(chan, num) })};
```
But this doesn't:
```
schedmidi.value({ midiout.program(chan, num) })}; donecmd.value(\program);
```
The call of `donecmd` seemed to be delayed in the latter case?

Reply:
`schedmidi` is delayed and happens after `donecmd` in both cases.

Reply:
I meant in the latter example, the command ends up being sent twice.

Reply:
Anyway it works as-is, I think this is ok for merging